extensions/zenloop-post-purchase-survey/src/index.jsx

```diff
@@ -16,142 +17,46 @@ import {
   Link,
   Image,
   Separator,
-  useExtensionInput,
 } from "@shopify/post-purchase-ui-extensions-react";
 
-/**
- * Entry point for the `ShouldRender` Extension Point.
- */
-extend("Checkout::PostPurchase::ShouldRender", async ({ storage, inputData }) => {
-  const shop = inputData?.shop;
+const SURVEY_BASE_URL = "https://zenresponses.zenloop.com/";
+
+// Replace with production URL when deploying the app
+const APP_URL = "https://pas-offline-mile-juice.trycloudflare.com";
```

Comment:
NOTE: This needs to be changed to the production url serving the app. @alinaqi @50ajmalkhan , let me know what the prod url is and I can make the update in this PR.